compile.sh

```diff
@@ -2,6 +2,31 @@ clang -O3 c/code.c -o c/code
 go build -o go/code go/code.go
 javac jvm/code.java
 RUSTFLAGS="-Zlocation-detail=none" cargo +nightly build --manifest-path rust/Cargo.toml --release
-kotlinc -include-runtime kotlin/code.kt -d kotlin/code.jar
-#kotlinc-native -include-runtime kotlin/code.kt -d kotlin/code
 dart compile exe dart/code.dart -o dart/code --target-os=macos
+kotlinc-jvm -include-runtime kotlin/jvm/code.kt -d kotlin/jvm/code.jar
+kotlinc-native kotlin/native/code.kt -o kotlin/native/code
```

Comment:
For better runtime performance you can add `-opt` argument. For reference: [docs](https://kotlinlang.org/docs/compiler-reference.html#opt).